src/modules/iotjs_module_crypto.h

```diff
@@ -15,6 +15,13 @@
 #ifndef IOTJS_MODULE_CRYPTO_H
 #define IOTJS_MODULE_CRYPTO_H
 
+typedef enum {
```

Comment:
Do we need the enum values in the header? Also are they in any correlation with the ones in the crypto.js? If yes we should add a comment about it.